packages/bvaughn-architecture-demo/src/contexts/FocusContext.tsx

```diff
@@ -68,7 +68,7 @@ export function FocusContextRoot({ children }: PropsWithChildren<{}>) {
     }
 
     const timeoutId = setTimeout(() => {
-      client.loadRegion({ begin: range.begin.time, end: range.end.time }, duration);
+      client.requestFocusRange({ begin: range.begin.time, end: range.end.time });
```

Comment:
Frontend has screenshot tests that run against a pre-recorded set of API calls to minimize test flakiness from backend issues. (We used to have a lot of those.) Adding new API calls invalidates the pre-recorded test data and causes tests to fail.

The new code channels all API calls through the Replay "client". That's how we record (and replay) them. This change (`client.loadRegion` -> `client.requestFocusRange`) looks like it might introduce a _new_ call to some of the tests, which might explain the failure.

You can read more about those tests here if you're interested:
https://github.com/replayio/devtools/tree/main/packages/bvaughn-architecture-demo/playwright

Reply:
I pulled this branch down and ran it locally to confirm the failing tests. I noticed a few tests had outdated screenshots, and one test failed because of the reason mentioned above: "_should support setting focus range via the context menu_"

1. To fix the hard failure above, I followed [these steps](https://github.com/replayio/devtools/tree/main/packages/bvaughn-architecture-demo/playwright#updating-protocol-fixture-data) to update the protocol fixture data. (Basically I focused the test mentioned above and then ran `yarn test:update-fixture-data`).
2. To update the changed screenshots, I followed [these steps](https://github.com/replayio/devtools/tree/main/packages/bvaughn-architecture-demo/playwright#locally-to-update-snapshots). (Basically I ran `earthly +playwright-update-snapshots --HOST=host.docker.internal --push +deploy` in the "playwright" directory.) The resulting changes looked _wrong_ though, and I realized that this branch was based off of a broken revision in master (fixed by 2ceb3a70ada47ef81b614f3eab386cddb09a449d) so I rebased.

I force pushed to this branch with the updated fixtures JSON. I'd expect CI to pass now.